tdrs-backend/tdpservice/test/test_admin.py

```diff
@@ -80,3 +83,22 @@ def test_unauth_cant_access_admin(client):
     url = reverse("admin:index")
     response = client.get(url)
     assert response.status_code == status.HTTP_302_FOUND
+
+
+@pytest.mark.django_db
+def test_admin_users_displays_keys(client, admin_user):
+    """Test an authenticated admin_user sees the appropriate content."""
+    client.login(username=admin_user.username, password="test_password")
+    url = reverse("admin:users_user_change", args=(admin_user.id,))
+    response = client.get(url)
+    assert "Staff status" in response.rendered_content
+    assert "Superuser status" in response.rendered_content
+
+
+@pytest.mark.django_db
+def test_superuser_env_var_is_set():
+    """Test to make sure a valid superuser is in the env."""
+    superuser = os.environ.get('DJANGO_SU_NAME')
+    assert superuser is not None
+    assert User.objects.filter(username=superuser).exists()
```

Comment:
what's the difference between lines 102 and 103 in terms of what's being tested?

Reply:
101: assign variable `superuser` from the environment variable `DJANGO_SU_NAME`
102: assert that the variable `superuser` was assigned a value (it would not be if the environment variable wasn't defined)
103: assert that their is a user in the system with the username assigned to the `superuser` variable